packages/config/src/globals.ts

```diff
@@ -108,3 +108,14 @@ export const serverPath = (
 export const databasePath = `${serverPath}db/`
 export const iconUploaderPath = `${serverPath}saveIcon`
 export const hiddenPrefix = '_#_'
+
+// PouchDb paths
+export const allDbs = 'allDbs'
+
+// RavenDb paths
+export const rdbPrefix = 'ravendb/'
+export const socketPath = 'http://localhost:4000'
+export const replicate = 'replicate/'
+export const deletePath = 'delete/'
+export const dbSuffix = '.sqlite'
```

Comment:
In case if we will compleatly remove pouch db we dont need it, currently you can switch databases 
https://github.com/serge-web/serge/pull/1195/files#diff-66c5602cae11b6d25bdcb24fc3ff243a6a94346e857b2e5cd7211fc1101189c3R21

Reply:
Sure @lilitkarapetyan - but this constant is under the `RavenDb` section - it should be under the `PouchDb` section.